dragonchain/lib/faas.py

```diff
@@ -0,0 +1,46 @@
+import os
+import base64
+import json
+from typing import List, Dict, Optional, cast, Any
+
+import requests
+
+from dragonchain import exceptions
+
+FAAS_GATEWAY = os.environ.get("FAAS_LOGS_GATEWAY") or os.environ["FAAS_GATEWAY"]
+
+
+def get_faas_auth() -> str:
+    """Gets authorization to use OpenFaaS
+
+        Returns:
```

Comment:
Un-indent for correct docstring and is probably worth mentioning in the docstring that this returns an empty string if it can't find the openfaas secrets rather than throwing an exception